src/Compilers/Core/MSBuildTask/Microsoft.CSharp.Core.targets

```diff
@@ -70,6 +70,7 @@
           AdditionalFiles="@(AdditionalFiles)"
           AllowUnsafeBlocks="$(AllowUnsafeBlocks)"
           Analyzers="@(Analyzer)"
+          AnalyzerDependencies="@(AnalyzerDependency)"
```

Comment:
@agocke Should we be capturing Analyzers and AnalyzerDependencies as inputs (that is, in the Inputs attribute of the Target element, at the top of the file)? It really seems like we should...